<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStep">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>490</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>480</y>
      <width>80</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>100</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>180</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionExit"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuLV">
    <property name="title">
     <string>Level</string>
    </property>
    <addaction name="actionSelect"/>
    <addaction name="separator"/>
    <addaction name="actionLV1"/>
    <addaction name="actionLV2"/>
    <addaction name="actionLV3"/>
    <addaction name="actionLV4"/>
    <addaction name="actionLV5"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuLV"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLevel1">
   <property name="text">
    <string>Level1</string>
   </property>
  </action>
  <action name="actionLevel2">
   <property name="text">
    <string>Level2</string>
   </property>
  </action>
  <action name="actionLevel3">
   <property name="text">
    <string>Level3</string>
   </property>
  </action>
  <action name="actionLevel4">
   <property name="text">
    <string>Level4</string>
   </property>
  </action>
  <action name="actionLevel5">
   <property name="text">
    <string>Level5</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="text">
    <string>Select</string>
   </property>
  </action>
  <action name="actionLV1">
   <property name="text">
    <string>Level1</string>
   </property>
  </action>
  <action name="actionLV2">
   <property name="text">
    <string>Level2</string>
   </property>
  </action>
  <action name="actionLV3">
   <property name="text">
    <string>Level3</string>
   </property>
  </action>
  <action name="actionLV4">
   <property name="text">
    <string>Level4</string>
   </property>
  </action>
  <action name="actionLV5">
   <property name="text">
    <string>Level5</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
